bootstrap/terraform/variables.tf

```diff
@@ -16,7 +16,7 @@ variable "name" {
 variable "cluster_version" {
   type        = string
   description = "Kubernetes Version"
-  default     = "1.30"
+  default     = "1.29"
```

Comment:
downgrading?

Reply:
It seems there were changes to CoreDNS in version 1.30 that have affected the ArgoCD deployment. The HAProxy isn't functioning as expected. I'm currently investigating the cause, but in the meantime, I think it's better to revert to the previous version. I'm also setting up end-to-end tests in a separate branch, which should make it easier for us to identify and resolve issues like that.